react/features/base/modal/components/JitsiScreen.js

```diff
@@ -62,24 +62,32 @@ const JitsiScreen = ({
     isModalPresentation = true,
     safeAreaInsets = [ 'left', 'right' ],
     style
-}: Props) => (
-    <View
-        style = { styles.jitsiScreenContainer }>
-        <JitsiKeyboardAvoidingView
-            contentContainerStyle = { contentContainerStyle }
-            hasBottomTextInput = { hasBottomTextInput }
-            hasTabNavigator = { hasTabNavigator }
-            isModalPresentation = { isModalPresentation }
-            style = { style }>
-            <SafeAreaView
-                edges = { safeAreaInsets }
-                style = { styles.safeArea }>
-                { children }
-            </SafeAreaView>
-            { footerComponent && footerComponent() }
-        </JitsiKeyboardAvoidingView>
-    </View>
-);
+}: Props) => {
+    // Tells the view what to do with taps
+    const shouldSetResponse = useCallback(() => true);
+    const onRelease = useCallback(() => Keyboard.dismiss());
+
+    return (
+        <View
+            onResponderRelease = { onRelease }
```

Comment:
Can't these props be set on the JitsiKeyboardAvoidingView ?

Reply:
Yes, we can.